### ISSUE TITLE
[Bug]: ComboBox Caret Button fails FastPass a11y audit

### ISSUE THREAD

**Issue by gfisher-vmsft** [microsoft/fluentui#26897]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
all browsers
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/v-grfisher/pen/wvEaOWq

## Actual Behavior
Multiple FastPass automated check failures on the Caret button in ComboBox
- *aria-allowed-role* ARIA role presentation is not allowed for given element
- *button-name* Ensures buttons have discernible text
- *presentation-role-conflict* Elements marked as presentational should not have global ARIA or tabindex to ensure all screen readers ignore them

## Expected Behavior
No FastPass automated check failures

### Requested priority

Low

### Products/sites affected

https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox

### Are you willing to submit a PR to fix?

yes

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
